src/templates/angular/src/app/pages/header/header.component.css

```diff
@@ -0,0 +1,80 @@
+<% if(isDark){%>
+.icon-switcher {
```

Comment:
may be we can use separate files for dark & light modes, & import according to users selection over theme.
like
header.component.light.css
header.component.dark.css

Reply:
'isDark' variable contains the Boolean value whether the theme template is selected or not. (not specifically to light or dark mode)

Reply:
@Chaman-Raghav-Inno we are updating the **isDark** with **isLightDarkTheme** in upcoming PR

Reply:
need changes